automatic/vscode-insiders.install/update.ps1

```diff
@@ -0,0 +1,45 @@
+import-module au
+import-module "$PSScriptRoot\..\..\extensions\chocolatey-core.extension\extensions\chocolatey-core.psm1"
+
+$releases32 = 'https://update.code.visualstudio.com/api/update/win32/insider/VERSION'
+$releases64 = 'https://update.code.visualstudio.com/api/update/win32-x64/insider/VERSION'
+
+if ($MyInvocation.InvocationName -ne '.') {
+  function global:au_BeforeUpdate {
+    $Latest.Checksum32 = Get-RemoteChecksum $Latest.URL32
+    $Latest.Checksum64 = Get-RemoteChecksum $Latest.URL64
+  }
+}
+
+function global:au_SearchReplace {
+    @{
+        'tools\chocolateyInstall.ps1' = @{
+            "(?i)(^\s*packageName\s*=\s*)('.*')" = "`$1'$($Latest.PackageName)'"
```

Comment:
This can be removed if you change the install script to use `$env:chocolateyPackageName`